### PR TITLE
Ex 13 - Pedro.

### PR DESCRIPTION
**Pull request by prdj**

### CODE REVIEW

File: 13-PB-language-boolean/index.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+//Converts the operands to the same type before making the comparison.
+// Yes, there is a difference between them. To serve different types of operands.
+/* console.log(3 = '3');
+              ^ */
+// ReferenceError: Invalid left-hand side in assignment. On equal sinal can only be used to define Varibles.
+
+
+console.log('2.')
+let exTwo = true;
+console.log(!exTwo);
```
Comment (Ungoot) [inline 13-PB-language-boolean/index.js:14]: Exercise wants you to use a ternary operator here.

Comment (prdj) [reply 13-PB-language-boolean/index.js:14]: Done. But i dont see it on github to do the pull request.

Comment (prdj) [reply 13-PB-language-boolean/index.js]: Done again xD

Comment (Ungoot) [reply 13-PB-language-boolean/index.js]: Thank you

**Pull request closed without merging**